packages/text-field/character-counter/README.md

```diff
@@ -0,0 +1,54 @@
+# React Text Field Character Counter
+
+MDC React Text Field Character Counter is a React Component which uses [MDC Text Field Character Counter](https://github.com/material-components/material-components-web/tree/master/packages/mdc-textfield/character-counter)'s CSS and foundation JavaScript.
+
+## Usage
+
+```js
+import CharacterCounter from '@material/react-text-field/character-counter/index.js';
+
+const MyComponent = () => {
+  return (
+    <CharacterCounter />
+  );
+}
+```
+
+## Props
+
+Prop Name | Type | Description
+--- | --- | ---
+className | String | CSS classes for element.
+template | String | You can set custom template. See below
+
+## Custom Template
+
+CharacterCounter provides customizing by setting the `template` prop in CharacterCounter.  
+If you contain `${count}` and `${maxLength}` in template, it will replace each value.  
+(${count} is replaced length of Input's value, ${maxLength} is replaced Input's maxLength)  
+For example, default template is `${count} / ${maxLength}`, so it appears `0 / 140`.  
+If you set template as `${count} : ${maxLength}`, it appears `0 : 140`.  
+
+### Sample
+
+``` js
+import React from 'react';
+import TextField, {CharacterCounter, Input} from '@material/react-text-field';
+
+class MyApp extends React.Component {
+  render() {
+    return (
+      <TextField characterCounter={<CharacterCounter template='${count} : ${maxLength}' />}>
+        <Input value={'Happy Coding!'} maxLength={140} />
```

Comment:
change to `<Input value={this.state.value} maxLength={140} onChange={(e) => this.setState({value: e.target.value})} />`.